### PR TITLE
Fix file to generate docs from meta + doc generation

### PR DESCRIPTION
**Pull request by KarishmaBothara**
This PR is fix the docs:meta script (MetadataMd) as per new metadata format.

updated

./scripts/MetadataMd.ts

updates the generated files
   
  ../docs/cennznet/attestation.md
          ../docs/cennznet/cennzx.md
          ../docs/cennznet/constants.md
          ../docs/cennznet/errors.md
          ../docs/cennznet/events.md
          ../docs/cennznet/extrinsics.md
          ../docs/cennznet/genericAsset.md
          ../docs/cennznet/governance.md
          ../docs/cennznet/nft.md
          ../docs/cennznet/rpc.md
          ../docs/cennznet/staking.md
          ../docs/cennznet/storage.md
          ../docs/interfaces/governance_types.deriveproposalinfo.md
          ../docs/interfaces/governance_types.proposal.md
          ../docs/interfaces/nft_types.collectioninfo.md
          ../docs/interfaces/nft_types.derivetokeninfo.md
          ../docs/interfaces/staking_types.derivestakingelected.md
          ../docs/interfaces/staking_types.derivestakingoverview.md
          ../docs/interfaces/staking_types.derivestakingquery.md
         ../docs/interfaces/staking_types.derivestakingstash.md
         ../docs/interfaces/staking_types.derivestakingwaiting.md
         ../docs/interfaces/types.derivebalancesaccount.md
         ../docs/interfaces/types.derivedsessionkeyinfo.md
         ../docs/interfaces/types.derivedstakinginfo.md
         ../docs/interfaces/types.estimatefeeparams.md
         ../docs/interfaces/types.paymentoptions.md

added new files

       ../docs/cennznet/eth20Peg.md
       ../docs/cennznet/ethBridge.md
       ../docs/interfaces/ethbridge_types.etheventproof.md

### CODE REVIEW

File: docs/cennznet/cennzx.md [hunk 1]
```diff
@@ -19,103 +19,125 @@ The following sections contain the module details.
...
 - **interface**: `api.query.cennzx.totalLiquidity`
 - **summary**:   Total liquidity holdings of all investors in an exchange. ie/ total_liquidity(exchange) == sum(liquidity_balance(exchange, user)) at all times 
  
 # Extrinsic
  
-### addLiquidity(asset_id: `Compact<AssetId>`, min_liquidity: `Compact<Balance>`, max_asset_amount: `Compact<Balance>`, core_amount: `Compact<Balance>`)
+### addLiquidity(asset_id: `Compact<u32>`, min_liquidity: `Compact<u128>`, max_asset_amount: `Compact<u128>`, core_amount: `Compact<u128>`)
 - **interface**: `api.tx.cennzx.addLiquidity`
-- **summary**:   Deposit core asset and trade asset at current ratio to mint liquidity Returns amount of liquidity minted. 
+- **summary**:   eposit core asset and trade asset at current ratio to mint liquidity eturns amount of liquidity minted.
```
Comment (KarishmaBothara) [inline docs/cennznet/cennzx.md:42]: need to check why it is eating up the first letter.. D from posit..R from returns for cennzx doc

File: docs/cennznet/cennzx.md [hunk 2]
```diff
@@ -19,103 +19,125 @@ The following sections contain the module details.
  
 # Storage
  
-### coreAssetId(): `AssetId`
+### coreAssetId(): `u32`
```
Comment (ken-centrality) [inline docs/cennznet/cennzx.md:22]: Does this matter if `AssetId` has been switched back to the generic `u32`?

File: docs/cennznet/cennzx.md [hunk 3]
```diff
@@ -19,103 +19,125 @@ The following sections contain the module details.
...
+- **summary**:   eposit core asset and trade asset at current ratio to mint liquidity eturns amount of liquidity minted. 
 
-  `origin` `asset_id` - The trade asset ID `min_liquidity` - The minimum liquidity to add `asset_amount` - Amount of trade asset to add `core_amount` - Amount of core asset to add 
+  origin` asset_id` - The trade asset ID min_liquidity` - The minimum liquidity to add asset_amount` - Amount of trade asset to add core_amount` - Amount of core asset to add 
  
-### buyAsset(recipient: `Option<AccountId>`, asset_to_sell: `Compact<AssetId>`, asset_to_buy: `Compact<AssetId>`, buy_amount: `Compact<Balance>`, maximum_sell: `Compact<Balance>`)
+### buyAsset(recipient: `Option<AccountId32>`, asset_to_sell: `Compact<u32>`, asset_to_buy: `Compact<u32>`, buy_amount: `Compact<u128>`, maximum_sell: `Compact<u128>`)
 - **interface**: `api.tx.cennzx.buyAsset`
-- **summary**:   Buy `asset_to_buy` with `asset_to_sell`. Caller specifies an exact `buy_amount` and a `maximum_sell` amount to pay. 
+- **summary**:   uy `asset_to_buy` with `asset_to_sell`. aller specifies an exact `buy_amount` and a `maximum_sell` amount to pay.
```
Comment (ken-centrality) [inline docs/cennznet/cennzx.md:48]: there are quite a few missing first letters scenarios btw

File: docs/cennznet/constants.md [hunk 4]
```diff
@@ -22,104 +28,135 @@ The following sections contain the module constants, also known as parameter typ
...
+- **interface**: `api.consts.grandpa.maxAuthorities`
+- **summary**:   Max Authorities in use 
 
 ___
 
 
 ## identity
  
-### basicDeposit: `BalanceOf`
+### basicDeposit: `29`
```
Comment (ken-centrality) [inline docs/cennznet/constants.md:74]: `29`?

**Review by ken-centrality — COMMENTED** [review]
Looks good, just a couple questions around weird type definition

File: docs/cennznet/cennzx.md (hunk 2 above)
Comment (KarishmaBothara) [reply docs/cennznet/cennzx.md:22]: https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/runtime/definitions.ts#L57
AssetId actually is u32.. This data comes from metadata

File: docs/cennznet/cennzx.md (hunk 3 above)
Comment (KarishmaBothara) [reply docs/cennznet/cennzx.md]: resolved

File: docs/cennznet/constants.md (hunk 4 above)
Comment (KarishmaBothara) [reply docs/cennznet/constants.md]: fixed

**Review by ken-centrality — APPROVED** [review]

**Pull request merged**